### PR TITLE
In segmented visitor log, allow opening segment editor in new tab w/ segment used.

### LINKED ISSUE #7215: In the Segmented Visitor Log, let me open in a new tab the Piwik UI with the custom segment applied
The goal of this issue is to improve the great Segmented Visitor Log feature (from #4633), and let users apply the Segment to all Piwik reports (rather than just see the segment applied to the Visitor Log popover).

For example, maybe we could add an icon or a textual link "Apply this segment to the report [new tab]". When clicked, a new tab will open and this custom segment will be applied to all reports. 

Thoughts?

### PR DESCRIPTION
**Pull request by diosmosis**
Todo:

* ~add UI test~ tried to add one but realized since it opens in a new tab, it would be rather difficult w/ current UI tests...

Fixes #7215

### CODE REVIEW

File: plugins/CoreHome/templates/_dataTableActions.twig [hunk 1]
```diff
@@ -103,6 +103,20 @@
             </a>
         {% endif %}
 
+        {% if properties.datatable_actions|default is not empty %}
+        {% for action in properties.datatable_actions %}
+            <a class='dataTableAction {{ action.id|e('html_attr') }}'
+               href='javascript:;' title="{{ action.title|e('html_attr') }}"
+            >
+                {% if action.icon starts with 'icon-' %}
+                    <span title="" class="{{ action.icon }}"></span>
```
Comment (sgiehl) [inline plugins/CoreHome/templates/_dataTableActions.twig]: any reason for not adding the title here? That prevents the tooltip from being displayed when hovering the icon directly.

**Review by sgiehl — CHANGES_REQUESTED** [review]
Works as expected. Left a minor comment

File: plugins/CoreHome/templates/_dataTableActions.twig (hunk 1 above)
Comment (diosmosis) [reply plugins/CoreHome/templates/_dataTableActions.twig]: No, probably a typo

**Comment by diosmosis** [comment]
Updated.

**Comment by sgiehl** [comment]
@diosmosis  actually guess it would make more sense to add `title="{{ action.title }}"` so the tooltip is visible correctly

**Comment by diosmosis** [comment]
@sgiehl It looks ok to me? Or do you mean visible in the code?

**Comment by sgiehl** [comment]
Hm. Works on Chrome on Linux as well. Need to check again on Windows, as it didn't work there last time...

**Comment by sgiehl** [comment]
Tested on windows again. Seems to work now in Chrome, FF and IE. Maybe it has been some browser caching issue.

**Review by sgiehl — APPROVED** [review]

**Pull request merged**